pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -475,6 +475,20 @@ self: super: {
       };
     });
 
+  vimacs = super.vimacs.overrideAttrs(old: {
+    buildPhase = ''
+      substituteInPlace bin/vim \
+        --replace '/usr/bin/vim' 'vim' \
```

Comment:
It's not quite that easy.  The way this is typically installed by a user (outside of Nix) is to add `$myscriptdir` to the user's PATH.  But inside the project's bin folder is `vim` and `gvim` so those would cause infinite recursion unless it knows the actual underlying path of the original vim installation.

(for more information, you can see here: https://github.com/andrep/vimacs/blob/master/doc/vimacs.txt)

Reply:
Fair enough. The nicer upstream solution would probably to filter itself out of `PATH` first and then call `vim`, but I understand that implementing that in bash would probably be a pain.